docs/release-notes.rst

```diff
@@ -1,24 +1,25 @@
 IPDK Kubernetes Infrastructure Offload Release Notes
-#############################################################
+====================================================
 
 IPDK 24.01
-************
+------------
 
 What's new in this Release
-===========================
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 - Service Load Balancing: Support for K8s Service of type ClusterIP.
-   Kubeproxy implementation is now offloaded to hardware.
-   Services can be created and dynamically distributed to endpoints.
-   For TCP, only the SYN packet goes through the load-balancing logic.
-   Entry is added to hardware CT table for treatment of subsequent packets.
-   Support for dynamic scale-up of endpoints.
+  Kubeproxy implementation is now offloaded to hardware.
+  Services can be created and dynamically distributed to endpoints.
+  For TCP, only the SYN packet goes through the load-balancing logic.
+  Entry is added to hardware CT table for treatment of subsequent packets.
+  Support for dynamic scale-up of endpoints.
```

Comment:
I thought these lines were updated to reflect caching for both TCP and UDP. These are somehow reverted?